src/en/anchira/src/eu/kanade/tachiyomi/extension/en/anchira/Anchira.kt

```diff
@@ -109,6 +112,12 @@ class Anchira : HttpSource(), ConfigurableSource {
             fetchMangaDetails(manga).map {
                 MangasPage(listOf(it), false)
             }
+        } else if (query.startsWith(SLUG_BUNDLE_PREFIX)) {
+            // bundle entries as chapters
+            val manga = SManga.create().apply { this.url = "?s=${query.substringAfter(SLUG_BUNDLE_PREFIX)}&order=1" }
```

Comment:
consider putting the filters as well (final url after filters are applied?)

Reply:
Last I tried with the "site's last iteration" adding other parameters didn't work that well so I went straight for the safe approach on this one.

The filters are also directly inside `searchMangaRequest` on this one, I'd have to put them on its own function

Reply:
Turns out it wasn't so hard since that whole function was basically only the filter already so managed to repurpose it for both the Fetch and Request functions